app/src/main/java/com/example/life_is_well/SettingsPageMain.kt

```diff
@@ -20,7 +20,7 @@ class SettingsPageMain : AppCompatActivity() {
 
     class SettingsFragment : PreferenceFragmentCompat() {
         override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
-            setPreferencesFromResource(R.xml.root_preferences, rootKey)
+            43ewsazxzsetPreferencesFromResource(R.xml.root_preferences, rootKey)
```

Comment:
```suggestion
            setPreferencesFromResource(R.xml.root_preferences, rootKey)
```

Reply:
double checked file is current on Mental Wellness branch and was able to push